### PR TITLE
Adding hesop SQL Scripts

### PR DESCRIPTION
**Pull request by Mandickel**

### CODE REVIEW

File: sql_scripts/1b_hesop_create.sql
Comment (adelmestri) [inline sql_scripts/1b_hesop_create.sql:1]: Even if in the data documentation tretspef and mainspef are reported as char(8), what the data contain are varchar(3). Please update

File: sql_scripts/1c_hesop_pk_idx_hesop_patient.sql
Comment (adelmestri) [inline sql_scripts/1c_hesop_pk_idx_hesop_patient.sql:1]: idx_hesop_patient_ethnicity should also include patid as first column

File: sql_scripts/4e_hesop_map_tbl_visit.sql
Comment (adelmestri) [inline sql_scripts/4e_hesop_map_tbl_visit.sql:1]: About the condition "WHERE t1.attended in (5);" add some comments so we can remember/understand what it means in practice

File: sql_scripts/1b_hesop_create.sql
Comment (Mandickel) [reply sql_scripts/1b_hesop_create.sql:1]: Fixed.

File: sql_scripts/1c_hesop_pk_idx_hesop_patient.sql
Comment (Mandickel) [reply sql_scripts/1c_hesop_pk_idx_hesop_patient.sql:1]: Fixed.

File: sql_scripts/1d_hesop_curation.sql
[the remark this answers is not in the document]
Comment (Mandickel) [reply sql_scripts/1d_hesop_curation.sql:1]: Fixed.

**Pull request merged**